### PR TITLE
Feature: update text size and color for the subscribe card

### LINKED ISSUE #24: Feature: update text size and color for the subscribe card
### Type of feature

🍕 Feature

### Current behavior

subscribe

### Suggested solution

- Change bg color to match the screenshot below (use `bg-slate-300`)
- change text size

subscribe_xd

### Additional context

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by dev-phantom**
## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] ⏩ Revert

## Description

I removed the former input and button on the subscriber card and added a new one with a better design
I also updated text size and color for the subscribe card
## Related Tickets & Documents

Fixed #24 
## Mobile & Desktop Screenshots/Recordings

image

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📕 storybook
- [x] 🙅 no documentation needed

### 📸 Credits

This pull request template was created by [Open Sauced](https://github.com/open-sauced) 🍕

### CODE REVIEW

**Comment by TAKANOME-DEV** [comment]
Hey @dev-phantom, Thanks for your contribution.
Since there is an issue for the footer and the `input + button` located in the subscribe section, I would suggest to revert those changes and only include the one mentioned in the issue #24

File: apps/client/src/pages/index.tsx [hunk 1]
```diff
@@ -167,18 +167,24 @@ export default function IndexPage() {
             {/* </div> */}
           </div>
         </section>
-        <section className="subscribe-section flex justify-center rounded-lg bg-slate-200 p-8 shadow">
+        <section className="subscribe-section flex justify-center rounded-xl bg-slate-300 p-8 shadow">
           <div className="flex max-w-2xl flex-col items-center justify-center gap-8 p-8">
```
Comment (TAKANOME-DEV) [inline apps/client/src/pages/index.tsx:171]:
```suggestion
          <div className="flex max-w-3xl flex-col items-center justify-center gap-8 p-8">
```

**Review by TAKANOME-DEV — CHANGES_REQUESTED** [review]
Minor adjustment here:

**Comment by dev-phantom** [comment]
@TAKANOME-DEV fixed

**Comment by TAKANOME-DEV** [comment]
Great, just resolve the conflicts in this PR and it will be ready to be merged

**Comment by dev-phantom** [comment]
Conflict Resolved

**Review by TAKANOME-DEV — APPROVED** [review]
Great :+1: Thanks for the contribution

**Pull request merged**